.github/rawWorkflows/gh-ci-parameterized-flow.txt

```diff
@@ -4,6 +4,10 @@
       matrix:
         jdk: [17]
     runs-on: ubuntu-latest
+    permissions:
+     id-token: write
+     contents: read
+     checks: write
```

Comment:
Do we still need this?

Reply:
I believe I do, one run failed without these https://github.com/majisourav99/venice/actions/runs/11675167928